ui/appui-react/src/appui-react/stagepanels/StagePanelDef.tsx

```diff
@@ -182,17 +216,17 @@ export class StagePanelDef extends WidgetHost {
 
     if (!config)
       return;
-    this._size = config.size;
+    this.size = config.size;
     this._defaultSize = config.size;
     this._maxSizeSpec = config.maxSize;
     this._minSize = config.minSize;
     if (config.defaultState !== undefined) {
-      this._panelState = config.defaultState;
       this._defaultState = config.defaultState;
+      this.panelState = config.defaultState;
```

Comment:
Assign to field to avoid firing change events when initializing?
```suggestion
      this._panelState = config.defaultState;
```

Reply:
One of unit tests was not passing due to `ninezonestate` not being updated after setting pinned state (through passing a config) and calling a getter. But this was only after my changes, originally the test expected to receive the `pinned` value from the internal state. So I am not sure if the test should be removed since it tests something else now (and undo my changes here), or leave it as it is now?